From f0c02f00bb4e77e78f12f3c8f69a3d6b5d2c8b73 Mon Sep 17 00:00:00 2001
From: Mervin Fansler <mervin.fansler@bric.ku.dk>
Date: Wed, 9 Apr 2025 12:57:12 +0200
Subject: [PATCH] move BiocManager to suggests

---
 DESCRIPTION | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/DESCRIPTION b/DESCRIPTION
index ee3a3f9..6574de2 100644
--- a/DESCRIPTION
+++ b/DESCRIPTION
@@ -6,7 +6,8 @@ Authors@R: c(person(c("Rafael", "A."), "Irizarry", email = "rafael_irizarry@dfci
                   role = c("aut", "cre")),
              person(c("Michael", "I."), "Love", role = "aut"))
 Depends: R (>= 3.1.2),
-Imports: RColorBrewer, BiocManager
+Imports: RColorBrewer
+Suggests: BiocManager
 Description: A series of shortcuts for routine tasks originally developed by
     Rafael A. Irizarry to facilitate data exploration.
 License: Artistic-2.0
